go.mod

```diff
@@ -123,14 +123,14 @@ require (
 	go.uber.org/mock v0.2.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
 	go.uber.org/zap v1.26.0 // indirect
-	golang.org/x/crypto v0.12.0 // indirect
```

Comment:
All these dependencies are indirect. We should consider leaving it to the upstream direct dependency to update their dependencies, otherwise we may introduce breaking changes.